<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="51" failures="0" name="com.yourcompany.SampleSauceTest" time="1,861.82" errors="13" skipped="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib"/>
    <property name="java.vm.version" value="25.45-b02"/>
    <property name="gopherProxySet" value="false"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="/Users/kristianmeier/Desktop/dev/travisci_localhost_demo"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/Users/kristianmeier/Desktop/dev/travisci_localhost_demo"/>
    <property name="java.runtime.version" value="1.8.0_45-b14"/>
    <property name="java.awt.graphicsenv" value="sun.awt.CGraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/endorsed"/>
    <property name="os.arch" value="x86_64"/>
    <property name="java.io.tmpdir" value="/var/folders/37/g13t538d5px2pdf_jhp6lfz80000gp/T/"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Mac OS X"/>
    <property name="classworlds.conf" value="/usr/local/Cellar/maven/3.3.3/libexec/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/Users/kristianmeier/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:."/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="10.10.5"/>
    <property name="http.nonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
    <property name="user.home" value="/Users/kristianmeier"/>
    <property name="user.timezone" value="America/Los_Angeles"/>
    <property name="java.awt.printerjob" value="sun.lwawt.macosx.CPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="kristianmeier"/>
    <property name="java.class.path" value="/usr/local/Cellar/maven/3.3.3/libexec/boot/plexus-classworlds-2.5.2.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher test"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.lwawt.macosx.LWCToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_45"/>
    <property name="java.ext.dirs" value="/Users/kristianmeier/Library/Java/Extensions:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java"/>
    <property name="sun.boot.class.path" value="/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/sunrsasign.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="/usr/local/Cellar/maven/3.3.3/libexec"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeBig"/>
    <property name="socksNonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
    <property name="ftp.nonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[15]" time="20.155"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[5]" time="86.319"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[13]" time="50.57"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[9]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:49 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:49 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:50 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:50 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:54 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:54 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:58 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:58 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:59 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:59 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:00 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:00 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[4]" time="72.583"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[1]" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[4]" time="49.047"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[7]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:45 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:45 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:47 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:47 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:49 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:49 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[6]" time="46.894"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[6]" time="54.116"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[10]" time="55.99"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[2]" time="73.784"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[13]" time="57.362"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[1]" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[9]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:28 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:28 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:36 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:36 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[0]" time="45.518"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[3]" time="28.808"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[12]" time="78.791"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[10]" time="44.47"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[9]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:21:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:08 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:13 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:13 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:30 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:30 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:31 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:31 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:31 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:31 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[6]" time="50.248"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[2]" time="23.081"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[11]" time="43.639"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[13]" time="44.717"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[12]" time="33.247"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[15]" time="50.052"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[7]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:28 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:28 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:29 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:35 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:36 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:36 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[3]" time="48.163"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[8]" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:21:16 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:16 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:17 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:17 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:51 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:51 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:51 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:51 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:55 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:55 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[5]" time="42.473"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[1]" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[14]" time="50.768"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[15]" time="38.246"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[8]" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:18 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:19 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:19 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[5]" time="54.006"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[0]" time="50.207"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[14]" time="62.957"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[3]" time="69.208"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[8]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:37 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:38 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:38 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:46 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:05 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:22:06 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:22:06 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[11]" time="59.213"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[0]" time="58.503"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[14]" time="18.59"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[2]" time="58.35"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[12]" time="53.43"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[11]" time="0.003">
    <error message="Cannot click on element (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 65 milliseconds
Build info: version: &apos;2.53.1&apos;, revision: &apos;a36b8b1&apos;, time: &apos;2016-06-30 17:37:03&apos;
System info: host: &apos;Sauce-Win7&apos;, ip: &apos;172.20.114.48&apos;, os.name: &apos;Windows 7&apos;, os.arch: &apos;x86&apos;, os.version: &apos;6.1&apos;, java.version: &apos;1.8.0_91&apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=9, platform=WINDOWS, proxy={proxyAutoconfigUrl=http://127.0.0.1:19876/pac.js, proxyType=pac}, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:8057/, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss}]
Session ID: XXXXXXXX-XXXX-XXXX-XXXX-XXXX25112a60
Command duration or timeout: 1.13 seconds
Build info: version: &apos;2.45.0&apos;, revision: &apos;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&apos;, time: &apos;2015-02-27 09:10:26&apos;
System info: host: &apos;Sauces-MacBook-Air-5.local&apos;, ip: &apos;172.20.10.4&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.10.5&apos;, java.version: &apos;1.8.0_45&apos;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{enablePersistentHover=true, ie.fileUploadDialogTimeout=3000, platform=WINDOWS, ie.ensureCleanSession=false, ie.browserCommandLineSwitches=, requireWindowFocus=false, takesScreenshot=true, browserName=internet explorer, javascriptEnabled=true, unexpectedAlertBehaviour=dismiss, browserAttachTimeout=0, ie.enableFullPageScreenshot=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, hasMetadata=true, version=9, proxy={proxyAutoconfigUrl=http://127.0.0.1:19876/pac.js, proxyType=pac}, nativeEvents=true, elementScrollBehavior=0, webdriver.remote.sessionid=8c2d7426e55c4efd9f068b1b3c9abc6e, initialBrowserUrl=http://localhost:8057/, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true}]
Session ID: 8c2d7426e55c4efd9f068b1b3c9abc6e" type="org.openqa.selenium.ElementNotVisibleException">org.openqa.selenium.ElementNotVisibleException: Cannot click on element (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 65 milliseconds
Build info: version: &apos;2.53.1&apos;, revision: &apos;a36b8b1&apos;, time: &apos;2016-06-30 17:37:03&apos;
System info: host: &apos;Sauce-Win7&apos;, ip: &apos;172.20.114.48&apos;, os.name: &apos;Windows 7&apos;, os.arch: &apos;x86&apos;, os.version: &apos;6.1&apos;, java.version: &apos;1.8.0_91&apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=9, platform=WINDOWS, proxy={proxyAutoconfigUrl=http://127.0.0.1:19876/pac.js, proxyType=pac}, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:8057/, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss}]
Session ID: XXXXXXXX-XXXX-XXXX-XXXX-XXXX25112a60
Command duration or timeout: 1.13 seconds
Build info: version: &apos;2.45.0&apos;, revision: &apos;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&apos;, time: &apos;2015-02-27 09:10:26&apos;
System info: host: &apos;Sauces-MacBook-Air-5.local&apos;, ip: &apos;172.20.10.4&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.10.5&apos;, java.version: &apos;1.8.0_45&apos;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{enablePersistentHover=true, ie.fileUploadDialogTimeout=3000, platform=WINDOWS, ie.ensureCleanSession=false, ie.browserCommandLineSwitches=, requireWindowFocus=false, takesScreenshot=true, browserName=internet explorer, javascriptEnabled=true, unexpectedAlertBehaviour=dismiss, browserAttachTimeout=0, ie.enableFullPageScreenshot=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, hasMetadata=true, version=9, proxy={proxyAutoconfigUrl=http://127.0.0.1:19876/pac.js, proxyType=pac}, nativeEvents=true, elementScrollBehavior=0, webdriver.remote.sessionid=8c2d7426e55c4efd9f068b1b3c9abc6e, initialBrowserUrl=http://localhost:8057/, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true}]
Session ID: 8c2d7426e55c4efd9f068b1b3c9abc6e
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:79)
	at com.yourcompany.SampleSauceTest.freeTrialTest(SampleSauceTest.java:216)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.openqa.selenium.ElementNotVisibleException: Cannot click on element (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 65 milliseconds
Build info: version: &apos;2.53.1&apos;, revision: &apos;a36b8b1&apos;, time: &apos;2016-06-30 17:37:03&apos;
System info: host: &apos;Sauce-Win7&apos;, ip: &apos;172.20.114.48&apos;, os.name: &apos;Windows 7&apos;, os.arch: &apos;x86&apos;, os.version: &apos;6.1&apos;, java.version: &apos;1.8.0_91&apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=9, platform=WINDOWS, proxy={proxyAutoconfigUrl=http://127.0.0.1:19876/pac.js, proxyType=pac}, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:8057/, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss}]
Session ID: XXXXXXXX-XXXX-XXXX-XXXX-XXXX25112a60
Build info: version: &apos;2.45.0&apos;, revision: &apos;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&apos;, time: &apos;2015-02-27 09:10:26&apos;
System info: host: &apos;Sauces-MacBook-Air-5.local&apos;, ip: &apos;172.20.10.4&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.10.5&apos;, java.version: &apos;1.8.0_45&apos;
Driver info: driver.version: RemoteWebDriver
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement$1.invoke(EventFiringWebDriver.java:335)
	at com.sun.proxy.$Proxy2.click(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement.click(EventFiringWebDriver.java:348)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.remote.server.KnownElements$1.invoke(KnownElements.java:64)
	at com.sun.proxy.$Proxy3.click(Unknown Source)
	at org.openqa.selenium.remote.server.handler.ClickElement.call(ClickElement.java:30)
	at org.openqa.selenium.remote.server.handler.ClickElement.call(ClickElement.java:1)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at org.openqa.selenium.remote.server.DefaultSession$1.run(DefaultSession.java:176)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</error>
    <system-out>SauceOnDemandSessionID=8c2d7426e55c4efd9f068b1b3c9abc6e job-name=freeTrialTest[11] : (Windows 7 internet explorer 9)
SauceOnDemandSessionID=8c2d7426e55c4efd9f068b1b3c9abc6e job-name=com.yourcompany.SampleSauceTest.freeTrialTest[11]
Job link: https://saucelabs.com/jobs/8c2d7426e55c4efd9f068b1b3c9abc6e?auth=9b8ea91b0c01a4f8dc928c33bf92c9d8
</system-out>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[10]" time="36.395"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[4]" time="39.209"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[7]" time="0.001">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at com.yourcompany.SampleSauceTest.tearDown(SampleSauceTest.java:233)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters.access$300(ConcurrentParameterized.java:170)
	at com.saucelabs.junit.ConcurrentParameterized$SauceClassRunnerForParameters$2.run(ConcurrentParameterized.java:287)
	at com.saucelabs.junit.ConcurrentParameterized$NonBlockingAsynchronousRunner$1.call(ConcurrentParameterized.java:312)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</error>
    <system-err>Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:20:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:40 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:41 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:41 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
Aug 18, 2016 1:21:42 PM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://ondemand.saucelabs.com:443: The target server failed to respond
Aug 18, 2016 1:21:42 PM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}-&gt;http://ondemand.saucelabs.com:443
</system-err>
  </testcase>
  <testcase classname="com.yourcompany.SampleSauceTest" name="loginTest[16]" time="49.105"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="verifyTitleTest[16]" time="28.65"/>
  <testcase classname="com.yourcompany.SampleSauceTest" name="freeTrialTest[16]" time="34.946"/>
</testsuite>